qa/tasks/cephfs/test_volume_client.py

```diff
@@ -53,16 +53,8 @@ def _sudo_write_file(self, remote, path, data):
 
         Both perms and owner are passed directly to chmod.
         """
-        remote.run(
-            args=[
-                'sudo',
-                'python',
-                '-c',
-                'import shutil, sys; shutil.copyfileobj(sys.stdin, file(sys.argv[1], "wb"))',
-                path,
-            ],
-            stdin=data,
-        )
+        remote.run(args=['sudo', self.py_version, '-c',
+                         'open("%s", "w").write("""%s""")' % (path, data)])
```

Comment:
I don't understand the need for this change. If anything, we shouldn't even be using python for this. could be this:

`remote.run(args=["sudo", "tee", path], stdin=data)`

Reply:
We can do that too. I was trying to make minimum changes to whatever is present in the file.